pkg/util/sys/device.go

```diff
@@ -205,7 +206,7 @@ func GetDeviceProperties(device string, executor exec.Executor) (map[string]stri
 
 // GetDevicePropertiesFromPath gets a device property from a path
 func GetDevicePropertiesFromPath(devicePath string, executor exec.Executor) (map[string]string, error) {
-	output, err := executor.ExecuteCommandWithOutput("lsblk", devicePath,
+	output, err := executor.ExecuteCommandWithOutput(60*time.Second, "lsblk", devicePath,
```

Comment:
This also comes back to my basic question... Do we really need to add timeouts in all these places? We know that ceph commands can hang because of network issues, but if we don't have reason to believe these other commands are hanging, there is no need to add the timeout. The timeouts may cause other issues if we don't get them right.